public/certificate.html

```
@@ -0,0 +1,278 @@
+</head>
+<body class="min-h-screen flex flex-col bg-white text-gray-900 dark:bg-black dark:text-gray-100 transition-colors duration-300">
+
+  <!-- ── SCREEN VIEW ─────────────────────────────────────────────────── -->
+  <div id="site-navbar"></div>
+
+  <main class="flex-1 max-w-6xl mx-auto w-full px-4 md:px-6 py-10 md:py-14">
+    <div class="flex items-center justify-between gap-4 mb-8">
+      <a href="/activity.html" class="inline-flex items-center gap-2 text-teal-600 dark:text-teal-400 font-semibold hover:underline">
+        <i class="fas fa-arrow-left"></i>Back to Activities
+      </a>
+      <button id="download-btn" class="inline-flex items-center gap-2 bg-gradient-to-r from-teal-500 to-cyan-500 hover:from-teal-600 hover:to-cyan-600 text-white font-bold px-5 py-3 rounded-xl shadow-lg hover:shadow-xl transition-all">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)